docs/evaluation/how_to_guides/datasets/export_filtered_traces_to_dataset.mdx

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 6
 ---
 
-# Export filtered traces from experiment to dataset
+# How to export filtered traces from experiment to dataset
```

Comment:
I'm not sure we gain anything by putting How-to in every title? These guides are already in the how-to section

Reply:
once you're on the page its not clear where you are bc sidebar doesn't expand that deep. personally like that langchain docs are all titled "How to..." but dont feel strongly